src/App.js

```diff
@@ -13,14 +13,13 @@ class App extends Component {
     this.score = {};
     this.keysInPractice = _.flatten(keyboardRows);
     this.state = {
-      currentKey: this.getRandomKey()
+      currentKey: this.getRandomKey(),
+      typedKey: ""
     };
   }
 
   getRandomKey = () => {
-    const thisKey = this.keysInPractice[
-      _.random(0, this.keysInPractice.length)
-    ];
+    const thisKey = _.sample(this.keysInPractice);
```

Comment:
every once in a while `undefined` would appear and you can't type anything to move on

Reply:
Thanks ^^^ I noticed the behavior, didn't find the bug. Can't believe I forgot about `_.sample, much better!